avalon/pipeline.py

```diff
@@ -31,6 +31,9 @@
 
 from .vendor import six
 
+# TODO acre needs to be in vendor
+import acre
```

Comment:
we should make this relative import 
`from .vendor import acre` and add it to vendor in avalon-core